library/common.py

```diff
@@ -25,6 +26,45 @@ def get_sequence_of_tokens(corpus):
             input_sequences.append(n_gram_sequence)
     return input_sequences, total_words
 
+
+# generate embeddings index
```

Comment:
I think this comment is redundant given the function name. What are we using our embedding index for? I think we are doing something similar to what they are doing [here](https://github.com/keras-team/keras/blob/7a39b6c62d43c25472b2c2476bd2a8983ae4f682/examples/pretrained_word_embeddings.py#L36).